### DISCUSSION TITLE
Is there a way to load fonts?

### DISCUSSION THREAD

**Discussion by mikepflu** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#314] [General]
I finally figured out where to run the fc-list command, but when I do, there are no fonts loaded.  Is there an easy way for me to load fonts into HA?

**Comment by PiotrMachowski** [comment]
I haven't found a way to do it, but to be honest I wasn't really trying either. I wanted to make it possible using python, but without any great success so far. Now that I have added a way to draw text on Lovelace card priority of this issue dropped significantly
